README_extending_docker.md

```diff
@@ -94,3 +94,21 @@ Lets consider two scenarios as starting point for extending Awesome docker
 ### How to create Custom Actions
 
 You can refer to [this link](https://docs.unskript.com/unskript-product-documentation/actions/create-custom-actions) on how to create custom Action
+
+
+## How to specify values for variables used in checks
+
+You can sepcify the variables that are used in the Checks in the Global file `unskript_config.yaml` You can see an example
+of that file in `unskript-ctl` Folder.  This step  be done before you start building your custom docker image.
+
+* In your `YOUR_REPO_DIRECTORY/actions/` Directory create a file unskript_config.yaml
```

Comment:
You use mix of upper case and lower case in the middle of your sentences, that doesnt sound right to me, why is Directory, and then you use folder up somewhere, can we be consistent, as this is external facing doc.